brainlets/girlsfrontline/static/girlsfrontline/js/sim.js

```diff
@@ -3357,6 +3361,7 @@ function simulateBattle() {
               dmg = dmg * Math.min(doll.battle.targets, enemy.count);
             }
           }
+          doll.shots += Math.floor ((doll.links - doll.battle.busylinks) * shotmultiplier * (doll.battle.acc / (doll.battle.acc + enemy.battle.eva)));
```

Comment:
Need to account for `hitCount` when `attackBuff !== undefined` is true and don't need to include `(doll.battle.acc / (doll.battle.acc + enemy.battle.eva)`